pages/k8s/release-notes.md

```diff
@@ -14,6 +14,30 @@ toc: False
 ---
 
 
+# 1.14 Bugfix release
+
+### April 23rd, 2019 - [canonical-kubernetes-???](https://api.jujucharms.com/charmstore/v5/canonical-kubernetes-???/archive/bundle.yaml)
```

Comment:
Canonical or Charmed?

Reply:
I'll suggest a middle ground. Charmonical.

Reply:
Thanks, changed to 'charmed'.